Runtime/BackgroundDownload.cs

```diff
@@ -110,7 +114,10 @@ public enum BackgroundDownloadStatus
     /// A class to perform file downloads that will continue if app goes into background or even gets shut down by OS.
     /// Note, that the download might not finish under specific conditions, for example Operating System can provide user with a feature to cancel such download.
     /// An instance of this class can be returned from the Coroutine to suspend it until download is finished.
-    /// The object of this class must be disposed when no longer required by calling Dispose().
+    /// The object of this class must be disposed when no longer required by calling Dispose() or by placing the object in the using() block.
+    /// If background download is disposed before completion, it is also cancelled, the existence and contents of result file is undefined.
+    /// The destination file must not be used before download completes. Otherwise it may prevent the download from writing to destination.
+    /// If app is quit by the operating system, on next run background downloads can be picked up by accessing BackgroundDownload.backgroundDownloads.
```

Comment:
Clarifying questions: What do we mean by "the app is quit by the operating system"? Is that a crash, an OS-initiated quit, or something else? Also, is this the only scenario where downloads can be resumed on next run?

Reply:
OS-initiated quit. The scenario here is that user stops using the app and it goes to the background from where is can be brought back to foreground quickly. However, the operating system can shut the app down entirely, for example in case the device gets low on memory. Background downloads are supposed to continue in such cases.
If the app crashes or is explicitly killed by user, there is no guarantee that background downloads will continue.

Reply:
Cool. Just wanted to make sure that this is the one specific scenario :)